BeerOverflowWindowsApp/GetBarList.cs

```diff
@@ -0,0 +1,69 @@
+﻿using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Data;
+using System.Drawing;
+using System.Linq;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows.Forms;
+using static TestProject.DataModels.GeodataDataModel;
+
+namespace BeerOverflowWindowsApp
+{
+    public partial class GetBarList : Form
+    {
+        const string GoogleAPILink = "https://maps.googleapis.com/maps/api/place/nearbysearch/json?location={0},{1}&radius={2}&type=bar&key=AIzaSyBqe4VYJPO86ui1aOtmpxapqwI3ET0ZaMY";
```

Comment:
You moved Google's API link to const string, but in method GetBarDataAsync there still is the link and not the variable.

Reply:
[Line 48: ](https://github.com/joklek/BeerOverflowWindowsApp/blob/fix_form1_refactor/BeerOverflowWindowsApp/GetBarList.cs#L48) `var response = await client.GetStringAsync(string.Format(GoogleAPILink, latitude, longitude, radius));`
The link is not there, or am I missing something?